lib/experimental/Widgets/Content/ProgressBarDuo/index.tsx

```diff
@@ -14,11 +13,8 @@ export const ProgressBarDuo = forwardRef<HTMLDivElement, DualProgressBarProps>(
     return (
       <div ref={ref} className="flex h-2 w-full gap-0.5 overflow-hidden">
         <div
-          className={cn(
-            "rounded transition-all duration-300 ease-in-out",
-            `bg-[${autoColor(0)}]`
```

Comment:
We should avoid passing interpolated classnames like this as tailwind works at build time and it won't recognise it when we're running it